tests/resources/Vsphere-VCH-Plugin.robot

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  This resource contains any keywords dealing with web based operations being performed within vSphere on the VCH plugin
+
+*** Keywords ***
+Navigate To Summary Tab
+    Wait Until Element Is Visible  css=ul.nav.nav-tabs > li:nth-child(1)
+    Wait Until Element Is Enabled  css=ul.nav.nav-tabs > li:nth-child(1)
```

Comment:
Instead of repeating `Wait Until Element Is Visible` and `Wait Until Element Is Enabled` everywhere, I would suggest creating a keyword `Wait for Element` and use it instead. This way it will be much easier in maintenance as well.

Reply:
Only saves us 1 line... not sure it is worth reducing the clarity of what we are doing for that.

Reply:
1 line x times its repeated. Its not just about that but also the maintenance required when you need to update those later. If its our keyword, then we can update / modify it as we need quickly.

Reply:
done